src/plugins/custom_import_map/public/components/vector_upload_options.tsx

```diff
@@ -50,11 +52,25 @@ const VectorUploadOptions = (props: RegionMapOptionsProps) => {
 
   const onChange = (files) => {
     if (files[0]) {
+      validateGeojsonFileFormat(files);
       validateFileSize(files);
       setFileContent(files);
     }
   };
 
+  const validateGeojsonFileFormat = (files) => {
+    const allowedExtensions = ['.json', '.geojson'];
```

Comment:
Should we move this to some config file?

Reply:
Sure, as it is not accessed in multiple places had kept it here : ) Moving it to common folder.